usefulUtilities/spawnPointZone/entityResources/entityClientScripts/spawnPointZone.js

```diff
@@ -34,21 +36,34 @@
                 print("ERROR: COULD NOT GET ZONE USER DATA");
                 return;
             }
+            if (!spawnAreaProperties.position || spawnAreaProperties.position.length < NUMBER_OF_AXES 
+                || !spawnAreaProperties.dimensions || spawnAreaProperties.dimensions < NUMBER_OF_AXES) {
+                return;
+            }
             if (spawnAreaProperties.usernameWhitelist && Array.isArray(spawnAreaProperties.usernameWhitelist)) {
                 (spawnAreaProperties.usernameWhitelist).forEach(function (newUsername) {
```

Comment:
can you please add `toLowerCase()` on this string comparison so that the whitelist is slightly easier to use?